EMS/Program.cs

```diff
@@ -0,0 +1,304 @@
+
+        _logger.LogInfo("Enter Location:");
+        string locationInput = Console.ReadLine();
+        if (Enum.TryParse(locationInput, true, out Location location))
+        {
+            employeeFilterObject.Location = location;
+        }
+        else
+        {
+            Console.WriteLine("Invalid location input!");
+        }
+
+        _logger.LogInfo("Enter JobTitle:");
+        string jobTitleInput = Console.ReadLine();
+        if (Enum.TryParse(jobTitleInput, true, out JobTitle jobTitle))
+        {
+            employeeFilterObject.JobTitle = jobTitle;
+        }
+        else
+        {
+            Console.WriteLine("Invalid location input!");
+        }
+
+        _logger.LogInfo("Enter Manager:");
+        employeeFilterObject.Manager = Console.ReadLine();
+
+
+        _logger.LogInfo("Enter Project:");
+        employeeFilterObject.Project = Console.ReadLine();
+        return employeeFilterObject;
+    }
+
+    private void Get(string? employeeNumber)
+    {
+        List<Employee> employees = _employeeDal.FetchData();
+        if (employees.Count > 0 && String.IsNullOrEmpty(employeeNumber))
+        {
+
+            foreach (var item in employees)
+            {
+                _logger.LogData("EmpNumber: {0}\n" + "EmpFirstName: {1}\n" + "EmpLastName: {2}\n" + "EmpDob: {3}\n" + "EmpEmailId: {4}\n" + "EmpMobileNumber: {5}\n" + "EmpJoiningDate: {6}\n" + "EmpLocation: {7}\n" + "EmpJobTitle: {8}\n" + "EmpDepartment: {9}\n" + "EmpManager: {10}\n" + "EmpProject: {11}\n", item.EmployeeNumber, item.FirstName, item.LastName, item.Dob, item.EmailId, item.MobileNumber, item.JoiningDate, item.Location, item.JobTitle, item.Department, item.Manager, item.Project);
+            }
+        }
+        else if (employees.Count > 0 && !string.IsNullOrEmpty(employeeNumber))
+        {
+            foreach (var item in employees)
+            {
+                if (item.EmployeeNumber == employeeNumber)
+                {
+                    _logger.LogData("EmpNumber: {0}\n" + "EmpFirstName: {1}\n" + "EmpLastName: {2}\n" + "EmpDob: {3}\n" + "EmpEmailId: {4}\n" + "EmpMobileNumber: {5}\n" + "EmpJoiningDate: {6}\n" + "EmpLocation: {7}\n" + "EmpJobTitle: {8}\n" + "EmpDepartment: {9}\n" + "EmpManager: {10}\n" + "EmpProject: {11}", item.EmployeeNumber, item.FirstName, item.LastName, item.Dob, item.EmailId, item.MobileNumber, item.JoiningDate, item.Location, item.JobTitle, item.Department, item.Manager, item.Project);
+                }
+            }
+        }
+        else
+        {
+            _logger.LogError("No Employee Data");
+        }
+    }
+
+    private void Help()
+    {
+        _logger.LogInfo("Options\n" + "add \t - \t To add an employee\n" + "display - \t To display all employee details\n" + "search \t - \t To display a particular employee data\n" + "delete \t - \t To delete an employee based on given employeenumber\n" + "update \t - \t To update employee details based on given employeenumber\n");
```

Comment:
Create methods for printing messages to console and use it instead of logger. Later if we pass file logger, your help will not work.